ext/hal/st/stm32cube/stm32f0xx/drivers/include/stm32f0xx_ll_spi.h

```diff
@@ -1385,7 +1385,7 @@ __STATIC_INLINE void LL_SPI_TransmitData8(SPI_TypeDef *SPIx, uint8_t TxData)
   */
 __STATIC_INLINE void LL_SPI_TransmitData16(SPI_TypeDef *SPIx, uint16_t TxData)
 {
-  *((__IO uint16_t *)&SPIx->DR) = TxData;
+  SPIx->DR = TxData;
 }

```

Comment:
Please document this patch in stm32f0xx/README (cf stm32l4xx/README)

Reply:
Ok